### PR TITLE
Feature: Raise pyblish error when subprocess fails

### PR DESCRIPTION
**Pull request by Tilix4**
## Changelog Description
Raises an error which stops pyblish if exception in make_paths_relative.

## Testing notes:
1. Fake a `raise SystemExit("blabla")` in `openpype\hosts\blender\utility_scripts\make_paths_relative.py` or `openpype/hosts/blender/utility_scripts/update_representations.py`
2. Publish anything

### CODE REVIEW

File: openpype/hosts/blender/utility_scripts/make_paths_relative.py [hunk 1]
```diff
@@ -23,8 +24,8 @@
                     str(Path(datablock.filepath).resolve()),
                     start=str(Path(bpy.data.filepath).parent.resolve()),
                 )
-        except (RuntimeError, ReferenceError, ValueError, OSError) as e:
-            log.error(e)
+        except BaseException as e:
+            raise SystemExit(e)
 
     bpy.ops.file.make_paths_relative()
     bpy.ops.wm.save_mainfile()
```
Comment (kaamaurice) [inline openpype/hosts/blender/utility_scripts/make_paths_relative.py]: Can we include this two final ops in a try except as above ?

**Review by kaamaurice — CHANGES_REQUESTED** [review]

File: openpype/hosts/blender/utility_scripts/make_paths_relative.py (hunk 1 above)
Comment (Tilix4) [reply openpype/hosts/blender/utility_scripts/make_paths_relative.py]: Sure!

File: openpype/hosts/blender/plugins/publish/integrate_blend.py [hunk 2]
```diff
@@ -118,12 +118,9 @@ def callback(id: ObjectId, future: Future):
 
                 # Submit command to pool
                 f = pool.submit(
-                    subprocess.check_call, main_command, shell=False
+                    subprocess.check_output, main_command, shell=False, stderr=subprocess.PIPE
```
Comment (Sharkitty) [inline openpype/hosts/blender/plugins/publish/integrate_blend.py]: Line too long

File: openpype/hosts/blender/plugins/publish/unpause_sync_server.py [hunk 3]
```diff
@@ -17,9 +93,37 @@ def process(self, context):
...
+                            f"Blend file: {blend_file[1]}\n\n{stack[0]}"
+                        )
+                    subprocess_errors = True
+                else:
+                    self.log.info(result)
+
+        # Stop if errors
+        if subprocess_errors:
+            raise RuntimeError(
+                "Errors happened during subprocesses. See above."
```
Comment (Sharkitty) [inline openpype/hosts/blender/plugins/publish/unpause_sync_server.py]:
```suggestion
                "Errors occured during subprocesses. See above."
```
I know, I'm picky, but it sounds better that way :ok_hand:

**Review by Sharkitty — CHANGES_REQUESTED** [review]

**Review by Sharkitty — APPROVED** [review]

**Pull request merged**